src/DistortableCollection.js

```diff
@@ -199,6 +199,19 @@ L.DistortableCollection = L.FeatureGroup.extend({
     return reduce / imgs.length;
   },
 
+  isJsonDetected(currentURL) {
+    if (currentURL.includes('?json=')) {
+      startIndex = currentURL.lastIndexOf('.');
```

Comment:
We may have to change this if we add more url parameters.

Reply:
Yes, but if we maintain the format where the shareable URLs always have `?json` at the initial part and `.json` at the ending part then there will be no need to change the code no matter the number of new parameters added to the URL.